resources/shellHelperFunctions.sh

```diff
@@ -137,8 +137,15 @@ write_download_info() {
 
   FILE_EXTENSION="${BASE_FILE##*.}"
 
-  FILE_SIZE=$(/usr/bin/stat -f"%z" "${BASE_PATH}/${BASE_FILE}")
-  MD5_HASH=$(md5 -q "${BASE_PATH}/${BASE_FILE}")
+  builder_echo "BUILDER_OS: ${BUILDER_OS}"
+  # stat flags to get filesize in bytes
+  if [ "$BUILDER_OS" == "mac" ] && [[ $(which stat) == /usr/bin/stat ]]; then
```

Comment:
hmm. @sgschantz says on his dev environment, `which stat` returns

> /opt/homebrew/opt/coreutils/libexec/gnubin/stat

And presumably the macOS build agents will have the same. This means, the iOS build goes into the else clause and stat doesn't expect to have `-c`.

This is even though the actual call is /usr/bin/stat (l. 147).
Shawn reported the version there 9.5 matched opt/homebrew/ :confused:

Reply:
OK, I didn't see this comment because it was on an outdated commit. Yes, we'd expect it to go into the `else` clause!